CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+* [Enhancement] Support latest versions of boto3; include the `request_checksum_calculation` setting for s3 client.
```

Comment:
I don't think it's particularly important what the internal configuration option name for the S3 client is; rather, the changelog should reflect the new config option that *this plugin* supports. Thus:
```suggestion
* [Enhancement] Support latest versions of boto3; include the `BACKUP_S3_REQUEST_CHECKSUM_CALCULATION` configuration option.
   Defaults to the value of `S3_REQUEST_CHECKSUM_CALCULATION` as set by the `s3` plugin.
```